step.yml

```diff
@@ -85,7 +85,12 @@ inputs:
 - pull_request_repository_url: $BITRISEIO_PULL_REQUEST_REPOSITORY_URL
   opts:
     category: Clone arguments
-    title: Pull request git URL
+    title: Pull request source repository
+    summary: URL of the source repository of a pull request.
+    description: |-
+      URL of the source repository of a pull request.
+
+      This points to the fork repository in builds triggered by pull requests.
```

Comment:
Could you run the readme generation workflow:
https://github.com/bitrise-steplib/steps-git-clone/blob/b9bccd63e0a32c85c7c568acdb00c492b0559f63/bitrise.yml#LL33

So this newly added description appears in the readme because at the moment it is empty:
https://github.com/bitrise-steplib/steps-git-clone/blob/b9bccd63e0a32c85c7c568acdb00c492b0559f63/README.md?plain=1#L61

Reply:
Thank you for reminding me!

Readme generator was broken due to a lot of missing `description` fields, so I a couple more modifications in my latest commit.